ngraph/core/reference/include/ngraph/runtime/reference/mean.hpp

```diff
@@ -21,44 +22,43 @@ namespace ngraph
         namespace reference
         {
             template <typename T>
-            void mean(const T* arg,
-                      T* out,
-                      const Shape& in_shape,
-                      const AxisSet& reduction_axes,
-                      bool keep_dims)
+            void mean(const T* arg, T* out, const Shape& in_shape, const AxisSet& reduction_axes)
             {
-                auto out_shape = reduce(in_shape, reduction_axes, keep_dims);
-                CoordinateTransform output_transform(out_shape);
-                std::vector<T> cs(shape_size(out_shape));
+                constexpr bool dont_keep_dims_in_output = false;
+                auto out_shape = reduce(in_shape, reduction_axes, dont_keep_dims_in_output);
```

Comment:
Do we need to have `keep_dims` in `reduce` reference implementation?

Reply:
Please notice that `dont_keep_dims_in_output ` is a constant expression evaluated as false, that is used to indicate that we do not keep dims when using `reduce` method (As dims with value 1 do not influence in index calculation)